src/readthedocs.yml

```diff
@@ -0,0 +1,8 @@
+build:
```

Comment:
This file should be sitting in the root directory. Since We already added this file in another merged PR, I'm going to remove this from this PR. @ambrosejcarr